sigmastate/src/main/scala/sigmastate/serialization/DataJsonEncoder.scala

```diff
@@ -46,7 +46,7 @@ object DataJsonEncoder {
   }
 
   private def encodeData[T <: SType](v: T#WrappedType, tpe: T): Json = tpe match {
-    case SUnit => Json.Null
+    case SUnit => Json.fromFields(mutable.WrappedArray.empty)
```

Comment:
Seems like JSON format has been changed?  Why? Why not covered in PR description?

Reply:
Yes, this fixes a bug with option serialization. 
The tests are added to DataJsonEncoderSpecification.
PR description updated.